src/test/ui/generic-associated-types/constraint-assoc-type-suggestion.rs

```diff
@@ -0,0 +1,17 @@
+// Test that correct syntax is used in suggestion to constrain associated type
+
+#![feature(generic_associated_types)]
```

Comment:
Out of curiosity, why not `#![allow(incomplete_features)]` in these? It's still one line and clears up the .stderr view.

Reply:
imo, `#![allow(incomplete_features)]` could be missed later on when this is no longer an incomplete feature. So, personally, I prefer to just have the warning in the stderr files now and have it removed later.